lib/iris/tests/results/usecases/pp_to_cf_conversion/to_netcdf/cf_checker/12187.b_0.txt

```diff
@@ -1,65 +0,0 @@
-
-CHECKING NetCDF FILE: /var/tmp/12187.b_0.nc
-=====================
-
-
-------------------
-Checking variable: level_height
-------------------
-
-------------------
-Checking variable: level_height_bnds
-------------------
-
-------------------
-Checking variable: model_level_number
-------------------
-
-------------------
-Checking variable: sigma_bnds
-------------------
-
-------------------
-Checking variable: forecast_period_bnds
-------------------
-
-------------------
-Checking variable: longitude
-------------------
-
-------------------
-Checking variable: latitude_longitude
-------------------
-
-------------------
-Checking variable: forecast_reference_time
-------------------
-
-------------------
-Checking variable: forecast_period
-------------------
-
-------------------
-Checking variable: time_bnds
-------------------
-
-------------------
-Checking variable: time
-------------------
-
-------------------
-Checking variable: latitude
-------------------
-
-------------------
-Checking variable: sigma
-------------------
-
-------------------
-Checking variable: tendency_of_upward_air_velocity_due_to_advection
-------------------
-ERROR (3.1): Units are not consistent with those given in the standard_name table.
```

Comment:
Perhaps there's not much point running cfchecker anyway if we're ignoring the errors! :see_no_evil: